modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for some connections within {product-title-short} and between {product-title-short} and external components are shown in the following table. You might need this information for configuring the firewalls in your system to allow traffic on certain ports.
+
+.{product-title-short} connections between components
+
+[cols="4",options="header"]
+|===
+| Component or external entity | Connection type | Port | Additional information
+
+| Central and Sensor on secured cluster
+a| * TCP/HTTPS
+* gRPC
+| 443
+| Sensor and Central primarily communicate over a bidirectional gRPC stream, initiated by Sensor to Central's port 443.
+
+
+| Central and user ({product-title-short} web portal or REST API)
+| HTTPS or gRPC
+| 443
+|
+
+| Central and user (CLI)
+| gRPC
+| 443
+|
+
+| Central and vulnerability feeds
+| HTTPS
+| 443
+|
+
+| Scanner and Scanner-DB
```

Comment:
Do we want to include Scanner V4 components, too? If so there are a few more:

Scanner V4 Indexer <-> Central (depends on the context)
Scanner V4 Indexer -> Scanner V4 DB (TCP 5432)
Scanner V4 Matcher <-> Central (depends on the context)
Scanner V4 Matcher -> Scanner V4 Indexer (gRPC I forget the port, but I can get it, if you want it)
Scanner V4 Matcher -> Scanner V4 DB (5432 TCP)